src/encoded/search.py

```diff
@@ -582,7 +582,7 @@ def format_facets(es_results, facets, used_filters, schemas, total, principals):
         if agg_name not in aggregations:
             continue
         terms = aggregations[agg_name][agg_name]['buckets']
-        if not any(bucket['doc_count'] > 0 for bucket in terms if 'doc_count' in bucket):
+        if not aggregations[agg_name]['doc_count'] > 0:
```

Comment:
We're using 
```
aggregations[agg_name]['doc_count']
```
twice so maybe set it to a variable
```
total = aggregations[agg_name]['doc_count']
```
Up to you.  I just find this way easier to read since we're adding it to a dict in the end.